tests/unit/components/Observations/ObsList.test.js

```diff
@@ -80,7 +82,7 @@ it( "renders multiple observations", async ( ) => {
 
 it( "renders grid view on button press", async ( ) => {
   await waitFor( ( ) => {
-    const { getByTestId } = renderComponent( <ObsList /> );
+    const { getByTestId } = renderComponentWithObsEditProvider( <ObsList /> );
```

Comment:
All of these tests pass for me with `renderComponent` What kind of errors were you seeing?

Reply:
Don't remember, but it seems to be working for me now too so I'll remove this.